packages/search-input/src/SearchResult/SearchResult.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { InputOption } from '@leafygreen-ui/input-option';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { getNodeTextContent } from '@leafygreen-ui/lib';
+import {
+  InferredPolymorphicProps,
+  Polymorphic,
+  PolymorphicAs,
+} from '@leafygreen-ui/polymorphic';
+
+import {
+  descriptionClassName,
+  descriptionStyle,
+  searchResultDisabledStyle,
+  searchResultStyles,
+  searchResultThemeStyles,
+  titleClassName,
+} from './SearchResult.styles';
+import { SearchResultProps } from './SearchResult.types';
+
+export const SearchResult = Polymorphic<
+  InferredPolymorphicProps<SearchResultProps>
+>(
+  (
+    {
+      as = 'li' as PolymorphicAs,
+      children,
+      description,
+      disabled,
+      className,
+      darkMode,
+      ...rest
+    },
+    ref,
+  ) => {
+    const { theme } = useDarkMode(darkMode);
+    const textContent = getNodeTextContent(children);
+    const ariaLabel =
+      rest['aria-label'] ?? rest['aria-labelledby'] ? '' : textContent;
```

Comment:
Shouldn't this be `rest['aria-label'] ?? (rest['aria-labelledby'] ??  textContent)`

Reply:
No, the logic is:

> If `aria-label` exists, use that; if `aria-labelledby` exists, then we have no label; if neither exist we set the label to `textContent`

Reply:
if i run this in the console `'hi' ?? undefined ? '' : 'hello'` `''` is printed. Is that correct?

Reply:
Ah. I think we need parens around the ternary: `'hi' ?? (undefined ? '' : 'hello')`. Not 100% sure why